tests/transformers/test_positional_encodings.py

```
@@ -0,0 +1,18 @@
+import torch
+
+from rtw_models.transformer.positional_encoding import SinusoidalPoistionalEncoding
+
+
+class TestPositionalEncodings:
```

Ruff (0.8.2)

Missing docstring in public class

(D101)